src/permissions/patterns.ts

```diff
@@ -29,8 +29,7 @@ export const SITES_PATTERN = "*://*/*";
  */
 export function getDomain(url: string): string {
   const urlClass = new URL(url);
-  const { domain } = parse(urlClass.hostname) as ParsedDomain;
-  return domain;
+  return psl.get(urlClass.hostname)!;
```

Comment:
```suggestion
	// The URL is already validated by `new URL` so the domain can't be empty
  return psl.get(urlClass.hostname)!;
```

I also simplified this code by using their own `.hostname` helper